concourse/pipeline.yml

```diff
@@ -204,10 +206,54 @@ jobs:
         on_success:
           <<: *health_status_notify
           params:
-            message: "Ggroup data to splunk validation dev deployed successfully."
+            message: "Ggroup data to splunk prod deployed successfully."
+            health: healthy
+        on_failure:
+          <<: *health_status_notify
+          params:
+            message: "Failed to deploy Ggroup data to splunk failed to deploy to prod."
+            health: unhealthy
+
+  - name: ggroup-data-to-splunk-prod-co-deploy
+    serial: true
+    plan:
+      - get: ggroup-data-to-splunk-git
+        trigger: true
+        passed: [ggroup-data-to-splunk-dev-deploy]
+      - task: terraform-prod-co-deploy
+        config:
+          inputs:
+          - name: ggroup-data-to-splunk-git
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: gdscyber/cyber-security-concourse-base-image
+              username: ((docker_hub_username))
+              password: ((docker_hub_password))
+          run:
+            path: /bin/bash
+            args:
+              - -c
+              - |
+                source /usr/local/bin/sts-assume-role.sh 'arn:aws:iam::779799343306:role/ggroup_data_to_splunk_concourse_role' 'eu-west-2'
+                  
+                pip install pipenv
+                cd ggroup-data-to-splunk-git/lambda
+                pipenv --python 3.7
+                pipenv install
+                
+                cd ..
+                export AWS_DEFAULT_REGION=eu-west-2
+                make zip
+                make terraform_apply_prod_co
+        on_success:
+          <<: *health_status_notify
+          params:
+            message: "Ggroup data to splunk prod co deployed successfully."
             health: healthy
         on_failure:
           <<: *health_status_notify
           params:
-            message: "Failed to deploy Ggroup data to splunk validation failed dev."
+            message: "Failed to deploy Ggroup data to splunk failed to deploy to prod co."
```

Comment:
It this a sentence?

Reply:
it doesn't look like one in hindsight. prod co here stands for prod cabinte office but I see how that's not clear

Reply:
wait I just reread that whole sentence and it's worse than I thought